platform/ios/resources/Base.lproj/Localizable.strings

```diff
@@ -20,7 +20,7 @@
 "DESIGNABLE" = "To display a Mapbox-hosted map here, set %1$@ to your access token in %2$@\n\nFor detailed instructions, see:";
 
 /* Setup documentation URL display string; keep as short as possible */
-"FIRST_STEPS_URL" = "mapbox.com/help/first-steps-ios-sdk";
+"FIRST_STEPS_URL" = "docs.mapbox.com/help/tutorials/first-steps-ios-sdk";
```

Comment:
This longer URL is less likely to fit in the designable. On the other hand, we no longer have a designable on iOS as of #13802. 🙈